.github/workflows/publish.yml

```diff
@@ -9,20 +9,21 @@ jobs:
   build-platform:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
I find this one handy so it doesn't stop as soon as it fails on one of the OS. I did the same for the test step below